localization/yabloc/yabloc_imgproc/include/yabloc_imgproc/lsd/lsd.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef YABLOC_IMGPROC__LSD__LSD_HPP_
+#define YABLOC_IMGPROC__LSD__LSD_HPP_
+
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Geometry>
+#include <opencv4/opencv2/core/eigen.hpp>
+#include <opencv4/opencv2/imgproc.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <sensor_msgs/msg/camera_info.hpp>
+#include <sensor_msgs/msg/compressed_image.hpp>
+#include <sensor_msgs/msg/image.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <optional>
+
+namespace yabloc::lsd
```

Comment:
I guess its better to make this file name something like "line_segment_detector" so that it will be self-contained

Reply:
I renamed it to line_segment_detector in  0eb5d5a